app-admin/opendoas/autobuild/build

```diff
@@ -0,0 +1,9 @@
+abinfo "Configure OpenDoas"
+"$SRCDIR"/configure --prefix=/usr \
```

Comment:
OpenDoas is not using autotools (no autotools files like configure.ac) and this would cause errors like "failed to reconfigure"

Reply:
> OpenDoas is not using autotools (no autotools files like configure.ac) and this would cause errors like "failed to reconfigure"

emm... Maybe you should refer to the logic of `/usr/lib/autobuild4/templates/10-autotools.sh` , my insight is that it looks for the existence of the configure.ac/configure/autogen.sh/bootstrap file at the same time and if it exists it proceeds with the

Reply:
tried `RECONF=0` in autobuild/defines, doesnt help.